<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="sportBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>391</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>运动</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>运动量/卡</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>次数/次</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="weightBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>重量/kg</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="timelabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>您好！</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_C">
    <property name="title">
     <string>操作(&amp;C)</string>
    </property>
    <addaction name="OpenAction"/>
    <addaction name="CloseAction"/>
    <addaction name="SaveAction"/>
    <addaction name="ClearAction"/>
    <addaction name="ExitAction"/>
   </widget>
   <addaction name="menu_C"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="OpenAction"/>
   <addaction name="CloseAction"/>
   <addaction name="SaveAction"/>
   <addaction name="ClearAction"/>
   <addaction name="ExitAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="OpenAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/src/open.png</normaloff>:/new/icon/src/open.png</iconset>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </action>
  <action name="CloseAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/src/stop.png</normaloff>:/new/icon/src/stop.png</iconset>
   </property>
   <property name="text">
    <string>关闭串口</string>
   </property>
  </action>
  <action name="SaveAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/src/save.png</normaloff>:/new/icon/src/save.png</iconset>
   </property>
   <property name="text">
    <string>保存数据</string>
   </property>
  </action>
  <action name="ClearAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/src/cleanport.png</normaloff>:/new/icon/src/cleanport.png</iconset>
   </property>
   <property name="text">
    <string>清空记录</string>
   </property>
  </action>
  <action name="ExitAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/src/exit.png</normaloff>:/new/icon/src/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
